CHANGELOG.md

```diff
@@ -17,7 +17,8 @@
 * [CHANGE] grpcutil: Convert Resolver into concrete type. #105
 * [CHANGE] grpcutil.Resolver.Resolve: Take a service parameter. #102
 * [CHANGE] grpcutil.Update: Remove gRPC LB related metadata. #102
-* [CHANGE] concurrency.ForEach: deprecated and reimplemented by new `concurrency.ForEachJob`. #113
+* [CHANGE] grpcclient: Bump default `grpc-max-send-msg-size` flag to 100 Mb. #123
+* [CHANGE] concurrency._ForEach_: deprecated and reimplemented by new `concurrency.ForEachJob`. #113
```

Comment:
Is the change on this line accidental?

Reply:
I take care of fixing this, so we can move forward with this PR.